core/src/main/java/org/jdbi/v3/core/argument/BeanPropertyArguments.java

```diff
@@ -21,18 +21,17 @@
 import java.lang.reflect.Method;
 import java.util.Optional;
 
-import org.jdbi.v3.core.argument.Argument;
-import org.jdbi.v3.core.argument.NamedArgumentFinder;
-import org.jdbi.v3.core.exception.UnableToCreateStatementException;
+import org.jdbi.v3.core.statement.StatementContext;
+import org.jdbi.v3.core.statement.UnableToCreateStatementException;
 
-class BeanPropertyArguments implements NamedArgumentFinder
+public class BeanPropertyArguments implements NamedArgumentFinder
```

Comment:
This logically belongs in the argument package, but I don't like exposing the constructor. Maybe a static factory method instead?